src/host/outputStream.cpp

```diff
@@ -849,46 +351,21 @@ bool ConhostInternalGetSet::IsVtInputEnabled() const
     return _io.GetActiveInputBuffer()->IsInVirtualTerminalInputMode();
 }
 
-// Method Description:
-// - Updates the buffer's current text attributes depending on whether we are
-//   starting/ending a hyperlink
-// Arguments:
-// - The hyperlink URI
-// Return Value:
-// - <none>
-void ConhostInternalGetSet::AddHyperlink(const std::wstring_view uri, const std::wstring_view params) const
-{
-    auto& screenInfo = _io.GetActiveOutputBuffer();
-    auto attr = screenInfo.GetAttributes();
-    const auto id = screenInfo.GetTextBuffer().GetHyperlinkId(uri, params);
-    attr.SetHyperlinkId(id);
-    screenInfo.GetTextBuffer().SetCurrentAttributes(attr);
-    screenInfo.GetTextBuffer().AddHyperlinkToMap(uri, id);
-}
-
-void ConhostInternalGetSet::EndHyperlink() const
-{
-    auto& screenInfo = _io.GetActiveOutputBuffer();
-    auto attr = screenInfo.GetAttributes();
-    attr.SetHyperlinkId(0);
-    screenInfo.GetTextBuffer().SetCurrentAttributes(attr);
-}
-
 // Routine Description:
-// - Replaces the active soft font with the given bit pattern.
+// - Lets accessibility apps know when an area of the screen has changed.
 // Arguments:
-// - bitPattern - An array of scanlines representing all the glyphs in the font.
-// - cellSize - The cell size for an individual glyph.
-// - centeringHint - The horizontal extent that glyphs are offset from center.
-// Return Value:
+// - changedRect - the area that has changed.
+// Return value:
 // - <none>
-void ConhostInternalGetSet::UpdateSoftFont(const gsl::span<const uint16_t> bitPattern,
-                                           const SIZE cellSize,
-                                           const size_t centeringHint)
+void ConhostInternalGetSet::NotifyAccessibilityChange(const til::rect changedRect)
 {
-    auto* pRender = ServiceLocator::LocateGlobals().pRender;
-    if (pRender)
+    auto& screenInfo = _io.GetActiveOutputBuffer();
+    if (screenInfo.HasAccessibilityEventing() && screenInfo.IsActiveScreenBuffer())

```

Comment:
The old `ConhostInternalGetSet::FillRegion` was notifying for the non-active screen buffer as well. I suppose this is a bug fix? (Because it does seem correct to not notify on the non-active one.)